sotodlib/io/imprinter_cli.py

```diff
@@ -79,6 +82,13 @@ def check_failed_books(imprint:Imprinter):
             imprint.bind_book(
                 book, ignore_tags=ignore_tags, ancil_drop_duplicates=ancil_drop_duplicates
             )
+            if imprint.docker_output_root is not None:
```

Comment:
I feel like you need a comment somewhere explaining what's going on with this docker_output_root business.  The behavior here seems a bit more complex than "Overide imprinter output root?"  Maybe it's simpler than I'm thinking, which is I might deserve some reassuring comment.